code/datums/status_effects/neutral.dm

```diff
@@ -460,6 +460,17 @@
 		QDEL_NULL(alt_clone)
 	return ..()
 
+// A status effect that represents a character crawling
+/datum/status_effect/crawling
+	id = "crawling"
+	duration = 6 SECONDS
+	status_type = STATUS_EFFECT_REFRESH
+	alert_type = null
+
+/datum/status_effect/crawling/on_remove()
+	if(owner.body_position == LYING_DOWN)
+		to_chat(owner, "<span class='notice'>You assume a prone posistion.")
```

Comment:
wherever you go with this you're gonna want this to be a balloon alert

Reply:
span class instead of span macros...

Reply:
I'm really unsure if this is important enough to be a balloon alert.  Should messages like "You stand up" and "You lay down" be popups?
I'm concerned that this information would be quite obtrusive to someone it is not relevant for.

Reply:
> span class instead of span macros...

oh man, this too, and actually "posistion" is spelled incorrected and the span doesn't even close lmao

> I'm really unsure if this is important enough to be a balloon alert. Should messages like "You stand up" and "You lay down" be popups? I'm concerned that this information would be quite obtrusive to someone it is not relevant for.

balloon alerts aren't for important things, they're actually for minor feedback that shouldn't clutter up the chat and keeping your eyes drawn towards the gameplay. The fact that they leave no log for you to review means important things actually should not be used for balloon alerts

Here's a hackmd made by my friend mothblocks with the relevant section on balloon alerts, it's a great read for understanding the use case
https://hackmd.io/@tgstation/SJIle5wYd#balloon_alertballoon_alert_to_viewers

Reply:
Alright, you've convinced me.